questions.md

```diff
@@ -0,0 +1,10 @@
+# Module 3 - Questions
+
+## Pull Requests
+1. Is it possible to limit permissions on a per branch basis?
+
+2. Who can perform a merge from a pull request?
+
+3. Is it possible to limit who you can recieve a pull request from?
+
+4. Is it possible to limit when you can recieve a pull request? (i.e. after a submission deadline)
```

Comment:
I assume this is for assessment? Here are a few scripts other teachers use:
* https://education.github.community/t/how-to-automatically-gather-or-collect-assignments/2595
* https://education.github.community/t/github-classroom-script-library/29878